packages/helm-charts/blockscout/templates/blockscout-migration.job.yaml

```diff
@@ -12,17 +12,24 @@ spec:
   template:
     spec:
       containers:
-      - name: blockscout-web
+      - name: blockscout-migration
         image: {{ .Values.blockscout.image.repository }}:{{ .Values.blockscout.image.tag }}
         imagePullPolicy: {{ .Values.imagePullPolicy }}
-        command: ["/bin/sh"]
-        args: ["-c", "echo Sleeping for 15; sleep 15; mix ecto.migrate"]
+        command:
+        - /bin/sh
+        - -c
+        args:
+        - |
+           [ ${DROP_DB} = "true" ] && mix do ecto.drop, ecto.create
+           mix do ecto.migrate
         resources:
           requests:
             memory: 250M
             cpu: 200m
         env:
-{{ include "celo.blockscout-env-vars" .  | indent 10 }}
+        - name: DROP_DB
+          value: "{{ default "false" .Values.blockscout.db.drop }}"
+{{ include "celo.blockscout-env-vars" .  | indent 8 }}
```

Comment:
Is this indent supposed to change?

Reply:
[Both syntax are valid](https://yaml.org/spec/1.2/spec.html#id2799784):
```
env:
- name: myvar
  value: myvalue
- name: myvar2
  value: myvalue2
```
or
```
env:
  - name: myvar
    value: myvalue
  - name: myvar2
    value: myvalue2
```

I changed to use the same indenting as we are using in this file for the other keys (like `containers`) but both should be valid as far as we indent `DROP_DB` with the same spaces.